WooCommerce/Classes/ViewRelated/Products/Edit Product/Edit Price/Bulk Edit Price/BulkUpdatePriceSettingsViewModel.swift

```diff
@@ -140,4 +155,57 @@ final class BulkUpdatePriceSettingsViewModel {
 
         return nil
     }
+
+    /// Returns the footer text to be displayed with information about the current bulk price and how many variations will be updated.
+    ///
+    private func footerText() -> String {
+        let numberOfVariations = bulkUpdateOptionsModel.productVariations.count
+        let numberOfVariationsText = String.pluralize(numberOfVariations,
+                                                      singular: Localization.variationsNumberSingularFooter,
+                                                      plural: Localization.variationsNumberPlurarFooter)
+
+        switch bulkUpdateOptionsModel.bulkValueOf(editingPriceType.keyPathForPriceType()) {
+        case .none:
+            return [Localization.currentPriceNoneFooter, numberOfVariationsText].joined(separator: " ")
+        case .mixed:
+            return [Localization.currentPriceMixedFooter, numberOfVariationsText].joined(separator: " ")
+        case let .value(price):
+            let currentPriceText = String.localizedStringWithFormat(Localization.currentPriceFooter, formatPriceString(price))
+            return [currentPriceText, numberOfVariationsText].joined(separator: " ")
+        }
+    }
```

Comment:
To not calculate this twice(Since it was calculated in the previews VC) I wonder if we should just provide the price type as a parameter 🤔

Reply:
We could but we are not storing the previous calculated result, so we would need to store it in the previous `BulkUpdateViewModel.swift` and updated it then the `BulkUpdateOptionsModel` changes (probably with a property observer), instead of doing this for all values(regular, sale price, dates etc.) do you think it would be better to simply cache the results in a opaque manner in `BulkUpdateOptionsModel` ?

Reply:
Nah, that is ok, no need to overcomplicate it.